src/adapter/src/coord.rs

```diff
@@ -1028,7 +1008,6 @@ impl Coordinator {
         // Ultimately, it doesn't concretely matter today, because the type ends up just being
         // u64 anyway.
         let mut policies_to_set: BTreeMap<Timestamp, CollectionIdBundle> = Default::default();
-        policies_to_set.insert(DEFAULT_LOGICAL_COMPACTION_WINDOW_TS, Default::default());
```

Comment:
I cannot remember how `policies_to_set` works. Could you give a TLDR of what the implications of this change is?

Reply:
When it was introduced (https://github.com/MaterializeInc/materialize/commit/9d2acaaf4462f1fb2597ea5c926b1d835e56cc66) we needed to insert this entry because code like https://github.com/MaterializeInc/materialize/commit/9d2acaaf4462f1fb2597ea5c926b1d835e56cc66#diff-c2e08eefc6ab97bcd6bcee678212d787fd838345cb4d04efd08498ac72e90607R523-R527 assumed the default compaction window already existed in the map. That's since been changed so we don't need this. 

The general policies_to_set is just a way to group everything by compaction window so that we can set a given compaction window for a set of objects at once